deploy/llms/Dockerfile.fastchat-worker-ray

```diff
@@ -0,0 +1,7 @@
+ARG BASE_IMAGE_VERSION="vllm-v0.1.0"
+FROM kubeagi/arcadia-fastchat-worker:${BASE_IMAGE_VERSION}
+
+# Overwrite the python version to pass the version check
```

Comment:
How this overwirte works❓

Reply:
the communication between ray head and worker will check the ray and python version, and it must match with x.y.z format, add more comments:
```
# Allow to use environment variable to set ray & python version to pass the version check
# for now, ray: 2.9.0, python: 3.9.x
# this utils.py file is from ray 2.9.0 ray-ml image
# search 'KubeAGI' in utils.py for what's changed
```